homework-submission/react-homework-w01/src/components/todoitem-static/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+//component for static list, I guess nothing much to see here :o
+
+class TodoListStatic extends Component {

```

Comment:
The name `TodoListStatic` suggests that this is a whole list, but actually it's only on list **item**. Try to find a better name :-)

Reply:
fixed since I destroyed the separate components to get a single one :)